src/test/java/core/basesyntax/service/impl/CsvFileReaderTest.java

```diff
@@ -0,0 +1,42 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.exceptions.NullFileException;
+import core.basesyntax.exceptions.ReadDataException;
+import core.basesyntax.service.FileReaderService;
+import java.io.File;
+import java.util.Collections;
+import java.util.List;
+import org.junit.Assert;
+import org.junit.Test;
+
+public class CsvFileReaderTest {
+    private static final List<String> TEST_DATA = List.of("test message1",
+            "aA1!@:',&*", "test message2");
+    private final File file = new File("src/test/resources/test.csv");
+    private final File emptyFile = new File("src/test/resources/empty.csv");
+    private final FileReaderService fileReader;
+
+    public CsvFileReaderTest() {
+        fileReader = new CsvFileReader();
+    }
+
+    @Test
+    public void readFromFile_Ok() {
```

Comment:
let's follow naming conventions for all test methods:
`<methodUnderTest>_<state>_<expectedBehavior>`